### PR TITLE
Fixed documentation bugs

### PR DESCRIPTION
**Pull request by mreiche**

### CODE REVIEW

File: doc/entities.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Creating an entity is simple.
 ```js
 import {Entity} from 'aurelia-orm';
 
+@resource()
```
Comment (RWOverdijk) [inline doc/entities.md]: Please add the string product

File: doc/api_entitymanager.md [hunk 2]
```diff
@@ -24,13 +24,17 @@ A `Repository` instance.
...
+import {UserEntity}    from 'entity/user';
 
 @inject(EntityManager)
 export class SomeUpdateViewModel {
   constructor (entityManager) {
     // Return the repository configured for user
     entityManager.getRepository('user');
+    
+    // Return the repository by Entity class
+    // That only works for Entities, that have the @repository() decorator
```
Comment (RWOverdijk) [inline doc/api_entitymanager.md]: This isn't true. This works for all entities. It just returns the default repository.

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: doc/api_entitymanager.md (hunk 2 above)
Comment (mreiche) [reply doc/api_entitymanager.md]: I file a bug then.

Comment (RWOverdijk) [reply doc/api_entitymanager.md]: There is no bug. This works as expected. What do you mean?

Comment (mreiche) [reply doc/api_entitymanager.md]: https://github.com/SpoonX/aurelia-orm/issues/302

**Comment by mreiche** [comment]
I added your change requests

**Review by RWOverdijk — APPROVED** [review]

**Comment by RWOverdijk** [comment]
Thanks!

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**